CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## Unreleased
+
+Refactoring the whole integration so it's platform agnostic.
+
+**BREAKING CHANGES:** users now must provide the `FullStory` client as an option in the args.
```

Comment:
So this will be `2.0.0` then?  Can we add a short migration snippet for users here?

We probably then need to update the README as well!

Reply:
I've updated the readme so we can merge this.